Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -545,8 +544,8 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
                 for (int j=djl; j<=depos_order+2-dju; j++) {
                     amrex::Real sdxi = 0._rt;
                     for (int i=dil; i<=depos_order+1-diu; i++) {
-                        sdxi += wqx*(sx_old[i] - sx_new[i])*((sy_new[j] + 0.5_rt*(sy_old[j] - sy_new[j]))*sz_new[k] +
-                                                             (0.5_rt*sy_new[j] + 1._rt/3._rt*(sy_old[j] - sy_new[j]))*(sz_old[k] - sz_new[k]));
+                        sdxi += wqx*(sx_old[i] - sx_new[i])*(0.5_rt*(sy_new[j] + sy_old[j])*sz_new[k] +
+                                                             (1._rt/6._rt*sy_new[j] + 1._rt/3._rt*sy_old[j])*(sz_old[k] - sz_new[k]));
```

Comment:
Could it be slightly more readable to write it like this? (I think it's fully equivalent in terms of number of operations from the compiler perspective, so it's just a question of whether the formulas -- here and below -- could be more readable this way or not really ...)
```suggestion
                                                             (sy_new[j]/6._rt + sy_old[j]/3._rt)*(sz_old[k] - sz_new[k]));
```

Reply:
Thanks for the suggestion. Actually, there is yet another PR (#2374) that makes this even more readable.